src/tests/JIT/Regression/JitBlue/Runtime_76194/CrossplatVirtualAlloc.cpp

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include <platformdefines.h>
+#include <stdint.h>
+#ifdef _WIN32
+#include <memoryapi.h>
+#else
+#include <sys/mman.h>
+#endif
+
+extern "C" DLL_EXPORT void* Alloc(uintptr_t size)
```

Comment:
```suggestion
extern "C" DLL_EXPORT void* AllocWithGuard(uintptr_t size)
```
This is not plain vanilla Alloc.